File: VB/RadioGroupControlToolTip/Form1.vb
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.Utils
Imports DevExpress.XtraEditors

Namespace RadioGroupControlToolTip

    Public Partial Class FormMain
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub toolTipController1_GetActiveObjectInfo(ByVal sender As Object, ByVal e As ToolTipControllerGetActiveObjectInfoEventArgs)
            Dim rg As RadioGroup = TryCast(e.SelectedControl, RadioGroup)
            If rg Is Nothing Then Return
            For i As Integer = 0 To rGroup.Properties.Items.Count - 1
                Dim rect As Rectangle = rGroup.GetItemRectangle(i)
                If rect.Contains(e.ControlMousePosition) Then
                    e.Info = New ToolTipControlInfo(i, rGroup.Properties.Items(i).Description)
                    Exit For
                End If
            Next
        End Sub
    End Class
End Namespace

File: CS/RadioGroupControlToolTip/Form1.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.Utils;
using DevExpress.XtraEditors;




namespace RadioGroupControlToolTip
{
    public partial class FormMain : Form
    {
        public FormMain()
        {
            InitializeComponent();
        }



        private void toolTipController1_GetActiveObjectInfo(object sender, DevExpress.Utils.ToolTipControllerGetActiveObjectInfoEventArgs e)
        {
            RadioGroup rg = e.SelectedControl as RadioGroup;
            if (rg == null) return;

            for (int i = 0; i < rGroup.Properties.Items.Count; i++)
            {
                Rectangle rect = rGroup.GetItemRectangle(i);
                
                if (rect.Contains(e.ControlMousePosition))
                {
                    e.Info = new ToolTipControlInfo(i, rGroup.Properties.Items[i].Description);
                    break;
                }
            }
        }
    }
}